docs/_posts/2014-02-24-community-roundup-17.md

```diff
@@ -41,7 +41,7 @@ Javier Aguirre ([@javaguirre](https://twitter.com/javaguirre)) put together a si
 
 ### SVG-based graphical node editor
 [NoFlo](http://noflojs.org/) and [Meemoo](http://meemoo.org/) developer [Forresto Oliphant](http://www.forresto.com/) built an awesome SVG-based [node editor](https://forresto.github.io/prototyping/react/) in React.
- <figure>[![](/react/img/blog/react-svg-fbp.png)](https://forresto.github.io/prototyping/react/)</figure>
+ [![](../img/blog/react-svg-fbp.png)](https://forresto.github.io/prototyping/react/)
```

Comment:
Just curious - why not use 'figure' tags for all of these images?

I am guessing we just want to avoid adding extra HTML to our markdown files as much as possible.